pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -783,3 +833,72 @@ func (a byPVCSize) Less(i, j int) bool {
 func claimToClaimKey(claim *v1.PersistentVolumeClaim) string {
 	return fmt.Sprintf("%s/%s", claim.Namespace, claim.Name)
 }
+
+// isCSIMigrationOnForPlugin checks if CSI migrartion is enabled for a given plugin.
+func isCSIMigrationOnForPlugin(pluginName string) bool {
+	switch pluginName {
+	case csiplugins.AWSEBSInTreePluginName:
+		return utilfeature.DefaultFeatureGate.Enabled(features.CSIMigrationAWS)
+	case csiplugins.GCEPDInTreePluginName:
+		return utilfeature.DefaultFeatureGate.Enabled(features.CSIMigrationGCE)
+	case csiplugins.AzureDiskInTreePluginName:
+		return utilfeature.DefaultFeatureGate.Enabled(features.CSIMigrationAzureDisk)
+	case csiplugins.CinderInTreePluginName:
+		return utilfeature.DefaultFeatureGate.Enabled(features.CSIMigrationOpenStack)
+	}
+	return false
+}
+
+// isPluginMigratedToCSI checks if an in-tree plugin has been migrated to a CSI driver on the node.
+func isPluginMigratedToCSI(pluginName string, csiNode *storagev1beta1.CSINode) bool {
```

Comment:
nit: will it be possible to name this something like `isPluginMigratedToCSIOnNode`?

I think there is a bit of logic here that is shared with [here](https://github.com/kubernetes/kubernetes/pull/83098/files#diff-bed922cbf0ba8772ff72617e43367c89R292) and we can refactor once the other PR lands.